test/e2e/healthcheck_run_test.go

```diff
@@ -54,6 +54,18 @@ var _ = Describe("Podman healthcheck run", func() {
 		Expect(hc).Should(Exit(125))
 	})
 
+	It("podman run healthcheck and logs should contain healthcheck output", func() {
+		session := podmanTest.Podman([]string{"run", "--name", "test", "-dt", "--health-interval", "1s", "--health-cmd", "'echo working'", "busybox"})
```

Comment:
```suggestion
		session := podmanTest.Podman([]string{"run", "--name", "test", "-dt", "--health-interval", "1s", "--health-cmd", "echo working", "busybox"})
```
you need to remove the extra quotes